### PR TITLE
Support query github pr with options

### PR DESCRIPTION
**Pull request by NullMDR**

### CODE REVIEW

File: lib/blobStorage.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
  */
 
 import { Aborter, AnonymousCredential, AppendBlobURL, BlockBlobURL, ContainerGetAccessPolicyResponse, ContainerURL, Credential, generateBlobSASQueryParameters, IBlobSASSignatureValues, Models, Pipeline, SASQueryParameters, ServiceURL, SharedKeyCredential, StorageURL } from "@azure/storage-blob";
-import { AppendBlobAppendBlockResponse, BlockBlobUploadResponse } from "@azure/storage-blob/typings/lib/generated/lib/models";
+import { AppendBlobAppendBlockResponse, BlockBlobUploadResponse } from "@azure/storage-blob/typings/src/generated/src/models";
```
Comment (raych1) [inline lib/blobStorage.ts]: What's the difference between the path of lib and src?

File: package-deps.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (raych1) [inline package-deps.json]: What is this file used for?

File: lib/blobStorage.ts (hunk 1 above)
Comment (NullMDR) [reply lib/blobStorage.ts]: For new version of @azure/storage-blob

File: package-deps.json (hunk 2 above)
Comment (NullMDR) [reply package-deps.json]: Removed

**Pull request merged**